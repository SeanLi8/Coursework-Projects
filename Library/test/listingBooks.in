add collection books.csv
add collection books2.csv
list all
list all long
list available 
list available long
number copies
add member Sean li
rent 100000 200001
rent 100000 200002
rent 100000 200003
rent 100000 200004
rent 100000 200005
rent 100000 200006
rent 100000 200007
rent 100000 200008
list available
list available long
list all 
list all long
exit