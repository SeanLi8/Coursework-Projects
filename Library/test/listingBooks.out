user: 26 books successfully added.

user: 8 books successfully added.

user: To Kill a Mockingbird (Harper Lee)
The Hobbit (J.R.R. Tolkien)
The Iliad (Homer)
The Odyssey (Homer)
The Hitchhiker's Guide to the Galaxy (Douglas Adams)
So Long and Thanks for all the Fish (Douglas Adams)
Batman Volume 1 (Bob Kane)
Harry Potter 1 (J.K. Rowling)
Harry Potter 2 (J.K. Rowling)
Harry Potter 3 (J.K. Rowling)
Harry Potter 4 (J.K. Rowling)
Harry Potter 5 (J.K. Rowling)
Harry Potter 6 (J.K. Rowling)
Wasteland (T.S. Eliot)
Biggles in the Battle (W.E. Johns)
Aeneid (Virgil)
Halloween (Jamie Lee Curtis)
Harry Potter 1 (J.K. Rowling)
Harry Potter 2 (J.K. Rowling)
Harry Potter 3 (J.K. Rowling)
Harry Potter 4 (J.K. Rowling)
Harry Potter 5 (J.K. Rowling)
Harry Potter 6 (J.K. Rowling)
To Kill a Mockingbird (Harper Lee)
To Kill a Mockingbird (Harper Lee)
To Kill a Mockingbird (Harper Lee)
Kaguya-sama wa Kokurasetai: Tensai-tachi no Renai Zunousen (Akasaka Aka)
Kaichou wa Maid-sama! (Fujiwara Hiro)
Bakuman (Obata Takeshi)
Shingeki no Kyojin (Isayama Hajime)
Hunter X Hunter (Togashi Yoshihiro)
Shokugeki no Somua (Saeki Shun)
Haikyuu!! (Furudate Haruichi)
Slam Dunk (Inoue Takehik0)

user: 111111: To Kill a Mockingbird (Harper Lee, Historical Fiction)
Currently available.

111112: The Hobbit (J.R.R. Tolkien, Fantasy)
Currently available.

111113: The Iliad (Homer, Poetry)
Currently available.

111114: The Odyssey (Homer, Poetry)
Currently available.

111115: The Hitchhiker's Guide to the Galaxy (Douglas Adams, Comedy)
Currently available.

111116: So Long and Thanks for all the Fish (Douglas Adams, Comedy)
Currently available.

111117: Batman Volume 1 (Bob Kane, Comic)
Currently available.

111118: Harry Potter 1 (J.K. Rowling, Fantasy)
Currently available.

111119: Harry Potter 2 (J.K. Rowling, Fantasy)
Currently available.

111120: Harry Potter 3 (J.K. Rowling, Fantasy)
Currently available.

111121: Harry Potter 4 (J.K. Rowling, Fantasy)
Currently available.

111122: Harry Potter 5 (J.K. Rowling, Fantasy)
Currently available.

111123: Harry Potter 6 (J.K. Rowling, Fantasy)
Currently available.

111124: Wasteland (T.S. Eliot, Poetry)
Currently available.

111125: Biggles in the Battle (W.E. Johns, War)
Currently available.

111126: Aeneid (Virgil, Epic)
Currently available.

111127: Halloween (Jamie Lee Curtis, Horror)
Currently available.

111128: Harry Potter 1 (J.K. Rowling, Fantasy)
Currently available.

111129: Harry Potter 2 (J.K. Rowling, Fantasy)
Currently available.

111130: Harry Potter 3 (J.K. Rowling, Fantasy)
Currently available.

111131: Harry Potter 4 (J.K. Rowling, Fantasy)
Currently available.

111132: Harry Potter 5 (J.K. Rowling, Fantasy)
Currently available.

111133: Harry Potter 6 (J.K. Rowling, Fantasy)
Currently available.

111134: To Kill a Mockingbird (Harper Lee, Historical Fiction)
Currently available.

111135: To Kill a Mockingbird (Harper Lee, Historical Fiction)
Currently available.

111136: To Kill a Mockingbird (Harper Lee, Historical Fiction)
Currently available.

200001: Kaguya-sama wa Kokurasetai: Tensai-tachi no Renai Zunousen (Akasaka Aka, Romantic Comedy)
Currently available.

200002: Kaichou wa Maid-sama! (Fujiwara Hiro, Romantic Comedy)
Currently available.

200003: Bakuman (Obata Takeshi, Shounen)
Currently available.

200004: Shingeki no Kyojin (Isayama Hajime, Shounen)
Currently available.

200005: Hunter X Hunter (Togashi Yoshihiro, Shounen)
Currently available.

200006: Shokugeki no Somua (Saeki Shun, Shounen)
Currently available.

200007: Haikyuu!! (Furudate Haruichi, Sports)
Currently available.

200008: Slam Dunk (Inoue Takehik0, Sports)
Currently available.

user: To Kill a Mockingbird (Harper Lee)
The Hobbit (J.R.R. Tolkien)
The Iliad (Homer)
The Odyssey (Homer)
The Hitchhiker's Guide to the Galaxy (Douglas Adams)
So Long and Thanks for all the Fish (Douglas Adams)
Batman Volume 1 (Bob Kane)
Harry Potter 1 (J.K. Rowling)
Harry Potter 2 (J.K. Rowling)
Harry Potter 3 (J.K. Rowling)
Harry Potter 4 (J.K. Rowling)
Harry Potter 5 (J.K. Rowling)
Harry Potter 6 (J.K. Rowling)
Wasteland (T.S. Eliot)
Biggles in the Battle (W.E. Johns)
Aeneid (Virgil)
Halloween (Jamie Lee Curtis)
Harry Potter 1 (J.K. Rowling)
Harry Potter 2 (J.K. Rowling)
Harry Potter 3 (J.K. Rowling)
Harry Potter 4 (J.K. Rowling)
Harry Potter 5 (J.K. Rowling)
Harry Potter 6 (J.K. Rowling)
To Kill a Mockingbird (Harper Lee)
To Kill a Mockingbird (Harper Lee)
To Kill a Mockingbird (Harper Lee)
Kaguya-sama wa Kokurasetai: Tensai-tachi no Renai Zunousen (Akasaka Aka)
Kaichou wa Maid-sama! (Fujiwara Hiro)
Bakuman (Obata Takeshi)
Shingeki no Kyojin (Isayama Hajime)
Hunter X Hunter (Togashi Yoshihiro)
Shokugeki no Somua (Saeki Shun)
Haikyuu!! (Furudate Haruichi)
Slam Dunk (Inoue Takehik0)

user: 111111: To Kill a Mockingbird (Harper Lee, Historical Fiction)
Currently available.

111112: The Hobbit (J.R.R. Tolkien, Fantasy)
Currently available.

111113: The Iliad (Homer, Poetry)
Currently available.

111114: The Odyssey (Homer, Poetry)
Currently available.

111115: The Hitchhiker's Guide to the Galaxy (Douglas Adams, Comedy)
Currently available.

111116: So Long and Thanks for all the Fish (Douglas Adams, Comedy)
Currently available.

111117: Batman Volume 1 (Bob Kane, Comic)
Currently available.

111118: Harry Potter 1 (J.K. Rowling, Fantasy)
Currently available.

111119: Harry Potter 2 (J.K. Rowling, Fantasy)
Currently available.

111120: Harry Potter 3 (J.K. Rowling, Fantasy)
Currently available.

111121: Harry Potter 4 (J.K. Rowling, Fantasy)
Currently available.

111122: Harry Potter 5 (J.K. Rowling, Fantasy)
Currently available.

111123: Harry Potter 6 (J.K. Rowling, Fantasy)
Currently available.

111124: Wasteland (T.S. Eliot, Poetry)
Currently available.

111125: Biggles in the Battle (W.E. Johns, War)
Currently available.

111126: Aeneid (Virgil, Epic)
Currently available.

111127: Halloween (Jamie Lee Curtis, Horror)
Currently available.

111128: Harry Potter 1 (J.K. Rowling, Fantasy)
Currently available.

111129: Harry Potter 2 (J.K. Rowling, Fantasy)
Currently available.

111130: Harry Potter 3 (J.K. Rowling, Fantasy)
Currently available.

111131: Harry Potter 4 (J.K. Rowling, Fantasy)
Currently available.

111132: Harry Potter 5 (J.K. Rowling, Fantasy)
Currently available.

111133: Harry Potter 6 (J.K. Rowling, Fantasy)
Currently available.

111134: To Kill a Mockingbird (Harper Lee, Historical Fiction)
Currently available.

111135: To Kill a Mockingbird (Harper Lee, Historical Fiction)
Currently available.

111136: To Kill a Mockingbird (Harper Lee, Historical Fiction)
Currently available.

200001: Kaguya-sama wa Kokurasetai: Tensai-tachi no Renai Zunousen (Akasaka Aka, Romantic Comedy)
Currently available.

200002: Kaichou wa Maid-sama! (Fujiwara Hiro, Romantic Comedy)
Currently available.

200003: Bakuman (Obata Takeshi, Shounen)
Currently available.

200004: Shingeki no Kyojin (Isayama Hajime, Shounen)
Currently available.

200005: Hunter X Hunter (Togashi Yoshihiro, Shounen)
Currently available.

200006: Shokugeki no Somua (Saeki Shun, Shounen)
Currently available.

200007: Haikyuu!! (Furudate Haruichi, Sports)
Currently available.

200008: Slam Dunk (Inoue Takehik0, Sports)
Currently available.

user: Aeneid (Virgil): 1
Bakuman (Obata Takeshi): 1
Batman Volume 1 (Bob Kane): 1
Biggles in the Battle (W.E. Johns): 1
Haikyuu!! (Furudate Haruichi): 1
Halloween (Jamie Lee Curtis): 1
Harry Potter 1 (J.K. Rowling): 2
Harry Potter 2 (J.K. Rowling): 2
Harry Potter 3 (J.K. Rowling): 2
Harry Potter 4 (J.K. Rowling): 2
Harry Potter 5 (J.K. Rowling): 2
Harry Potter 6 (J.K. Rowling): 2
Hunter X Hunter (Togashi Yoshihiro): 1
Kaguya-sama wa Kokurasetai: Tensai-tachi no Renai Zunousen (Akasaka Aka): 1
Kaichou wa Maid-sama! (Fujiwara Hiro): 1
Shingeki no Kyojin (Isayama Hajime): 1
Shokugeki no Somua (Saeki Shun): 1
Slam Dunk (Inoue Takehik0): 1
So Long and Thanks for all the Fish (Douglas Adams): 1
The Hitchhiker's Guide to the Galaxy (Douglas Adams): 1
The Hobbit (J.R.R. Tolkien): 1
The Iliad (Homer): 1
The Odyssey (Homer): 1
To Kill a Mockingbird (Harper Lee): 4
Wasteland (T.S. Eliot): 1

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: To Kill a Mockingbird (Harper Lee)
The Hobbit (J.R.R. Tolkien)
The Iliad (Homer)
The Odyssey (Homer)
The Hitchhiker's Guide to the Galaxy (Douglas Adams)
So Long and Thanks for all the Fish (Douglas Adams)
Batman Volume 1 (Bob Kane)
Harry Potter 1 (J.K. Rowling)
Harry Potter 2 (J.K. Rowling)
Harry Potter 3 (J.K. Rowling)
Harry Potter 4 (J.K. Rowling)
Harry Potter 5 (J.K. Rowling)
Harry Potter 6 (J.K. Rowling)
Wasteland (T.S. Eliot)
Biggles in the Battle (W.E. Johns)
Aeneid (Virgil)
Halloween (Jamie Lee Curtis)
Harry Potter 1 (J.K. Rowling)
Harry Potter 2 (J.K. Rowling)
Harry Potter 3 (J.K. Rowling)
Harry Potter 4 (J.K. Rowling)
Harry Potter 5 (J.K. Rowling)
Harry Potter 6 (J.K. Rowling)
To Kill a Mockingbird (Harper Lee)
To Kill a Mockingbird (Harper Lee)
To Kill a Mockingbird (Harper Lee)

user: 111111: To Kill a Mockingbird (Harper Lee, Historical Fiction)
Currently available.

111112: The Hobbit (J.R.R. Tolkien, Fantasy)
Currently available.

111113: The Iliad (Homer, Poetry)
Currently available.

111114: The Odyssey (Homer, Poetry)
Currently available.

111115: The Hitchhiker's Guide to the Galaxy (Douglas Adams, Comedy)
Currently available.

111116: So Long and Thanks for all the Fish (Douglas Adams, Comedy)
Currently available.

111117: Batman Volume 1 (Bob Kane, Comic)
Currently available.

111118: Harry Potter 1 (J.K. Rowling, Fantasy)
Currently available.

111119: Harry Potter 2 (J.K. Rowling, Fantasy)
Currently available.

111120: Harry Potter 3 (J.K. Rowling, Fantasy)
Currently available.

111121: Harry Potter 4 (J.K. Rowling, Fantasy)
Currently available.

111122: Harry Potter 5 (J.K. Rowling, Fantasy)
Currently available.

111123: Harry Potter 6 (J.K. Rowling, Fantasy)
Currently available.

111124: Wasteland (T.S. Eliot, Poetry)
Currently available.

111125: Biggles in the Battle (W.E. Johns, War)
Currently available.

111126: Aeneid (Virgil, Epic)
Currently available.

111127: Halloween (Jamie Lee Curtis, Horror)
Currently available.

111128: Harry Potter 1 (J.K. Rowling, Fantasy)
Currently available.

111129: Harry Potter 2 (J.K. Rowling, Fantasy)
Currently available.

111130: Harry Potter 3 (J.K. Rowling, Fantasy)
Currently available.

111131: Harry Potter 4 (J.K. Rowling, Fantasy)
Currently available.

111132: Harry Potter 5 (J.K. Rowling, Fantasy)
Currently available.

111133: Harry Potter 6 (J.K. Rowling, Fantasy)
Currently available.

111134: To Kill a Mockingbird (Harper Lee, Historical Fiction)
Currently available.

111135: To Kill a Mockingbird (Harper Lee, Historical Fiction)
Currently available.

111136: To Kill a Mockingbird (Harper Lee, Historical Fiction)
Currently available.


user: To Kill a Mockingbird (Harper Lee)
The Hobbit (J.R.R. Tolkien)
The Iliad (Homer)
The Odyssey (Homer)
The Hitchhiker's Guide to the Galaxy (Douglas Adams)
So Long and Thanks for all the Fish (Douglas Adams)
Batman Volume 1 (Bob Kane)
Harry Potter 1 (J.K. Rowling)
Harry Potter 2 (J.K. Rowling)
Harry Potter 3 (J.K. Rowling)
Harry Potter 4 (J.K. Rowling)
Harry Potter 5 (J.K. Rowling)
Harry Potter 6 (J.K. Rowling)
Wasteland (T.S. Eliot)
Biggles in the Battle (W.E. Johns)
Aeneid (Virgil)
Halloween (Jamie Lee Curtis)
Harry Potter 1 (J.K. Rowling)
Harry Potter 2 (J.K. Rowling)
Harry Potter 3 (J.K. Rowling)
Harry Potter 4 (J.K. Rowling)
Harry Potter 5 (J.K. Rowling)
Harry Potter 6 (J.K. Rowling)
To Kill a Mockingbird (Harper Lee)
To Kill a Mockingbird (Harper Lee)
To Kill a Mockingbird (Harper Lee)
Kaguya-sama wa Kokurasetai: Tensai-tachi no Renai Zunousen (Akasaka Aka)
Kaichou wa Maid-sama! (Fujiwara Hiro)
Bakuman (Obata Takeshi)
Shingeki no Kyojin (Isayama Hajime)
Hunter X Hunter (Togashi Yoshihiro)
Shokugeki no Somua (Saeki Shun)
Haikyuu!! (Furudate Haruichi)
Slam Dunk (Inoue Takehik0)

user: 111111: To Kill a Mockingbird (Harper Lee, Historical Fiction)
Currently available.

111112: The Hobbit (J.R.R. Tolkien, Fantasy)
Currently available.

111113: The Iliad (Homer, Poetry)
Currently available.

111114: The Odyssey (Homer, Poetry)
Currently available.

111115: The Hitchhiker's Guide to the Galaxy (Douglas Adams, Comedy)
Currently available.

111116: So Long and Thanks for all the Fish (Douglas Adams, Comedy)
Currently available.

111117: Batman Volume 1 (Bob Kane, Comic)
Currently available.

111118: Harry Potter 1 (J.K. Rowling, Fantasy)
Currently available.

111119: Harry Potter 2 (J.K. Rowling, Fantasy)
Currently available.

111120: Harry Potter 3 (J.K. Rowling, Fantasy)
Currently available.

111121: Harry Potter 4 (J.K. Rowling, Fantasy)
Currently available.

111122: Harry Potter 5 (J.K. Rowling, Fantasy)
Currently available.

111123: Harry Potter 6 (J.K. Rowling, Fantasy)
Currently available.

111124: Wasteland (T.S. Eliot, Poetry)
Currently available.

111125: Biggles in the Battle (W.E. Johns, War)
Currently available.

111126: Aeneid (Virgil, Epic)
Currently available.

111127: Halloween (Jamie Lee Curtis, Horror)
Currently available.

111128: Harry Potter 1 (J.K. Rowling, Fantasy)
Currently available.

111129: Harry Potter 2 (J.K. Rowling, Fantasy)
Currently available.

111130: Harry Potter 3 (J.K. Rowling, Fantasy)
Currently available.

111131: Harry Potter 4 (J.K. Rowling, Fantasy)
Currently available.

111132: Harry Potter 5 (J.K. Rowling, Fantasy)
Currently available.

111133: Harry Potter 6 (J.K. Rowling, Fantasy)
Currently available.

111134: To Kill a Mockingbird (Harper Lee, Historical Fiction)
Currently available.

111135: To Kill a Mockingbird (Harper Lee, Historical Fiction)
Currently available.

111136: To Kill a Mockingbird (Harper Lee, Historical Fiction)
Currently available.

200001: Kaguya-sama wa Kokurasetai: Tensai-tachi no Renai Zunousen (Akasaka Aka, Romantic Comedy)
Rented by: 100000.

200002: Kaichou wa Maid-sama! (Fujiwara Hiro, Romantic Comedy)
Rented by: 100000.

200003: Bakuman (Obata Takeshi, Shounen)
Rented by: 100000.

200004: Shingeki no Kyojin (Isayama Hajime, Shounen)
Rented by: 100000.

200005: Hunter X Hunter (Togashi Yoshihiro, Shounen)
Rented by: 100000.

200006: Shokugeki no Somua (Saeki Shun, Shounen)
Rented by: 100000.

200007: Haikyuu!! (Furudate Haruichi, Sports)
Rented by: 100000.

200008: Slam Dunk (Inoue Takehik0, Sports)
Rented by: 100000.

user: Ending Library process.
